Makefile

```diff
@@ -17,10 +22,42 @@ COMMON_ARGS += --frontend-opt build-arg:GOLANG_VERSION=$(GOLANG_VERSION)
 COMMON_ARGS += --frontend-opt build-arg:SHA=$(SHA)
 COMMON_ARGS += --frontend-opt build-arg:TAG=$(TAG)
 
-all: kernel initramfs rootfs osctl-linux-amd64 osctl-darwin-amd64 test lint docs installer
+all: buildkitd builddeps kernel initramfs rootfs osctl-linux-amd64 osctl-darwin-amd64 test lint docs installer
+
+VPATH = $(PATH)
+
+.PHONY: builddeps
+builddeps: gitmeta buildctl
+
+gitmeta:
+	GO111MODULE=off go get github.com/autonomy/gitmeta
+
+buildctl:
+	go get github.com/moby/buildkit/cmd/buildctl
+
+.PHONY: buildkitd
+buildkitd:
+ifeq ($(CONTAINERSTOPPED),$(BUILDKIT_CONTAINER))
+	@echo "Removing exited talos-buildkit container"
+	@docker rm $(BUILDKIT_CONTAINER)
+endif
+ifneq ($(CONTAINERRUNNING),$(BUILDKIT_CONTAINER))
+	@echo "Starting talos-buildkit container"
+	@docker run \
+		--name $(BUILDKIT_CONTAINER) \
+		-d \
+		--privileged \
+		 -p 1234:1234 \
+		$(BUILDKIT_IMAGE) \
+		--addr $(BUILDKIT_HOST)
+endif
+
+.PHONY: ci
+ci: builddeps buildkitd
 
 base:
-	@buildctl build \
+	@buildctl --addr $(BUILDKIT_HOST) \
```

Comment:
I think we can drop this if we leave the local build optional. The CLI supports the `BUILDKIT_HOST` env var. To reach my local machine I export this variable. After pulling this PR in and running it it looks like the env var takes precedence over the `--addr` flag.